### PR TITLE
Minor feature package

### PR DESCRIPTION
**Pull request by ghost**
- Suit storage units can now be repaired by using wirecutters, 5 cable, a crowbar, and 3 metal, in that order.
- Suit storage units have much less snowflakey code.
- Signs can now be unfastened with a wrench.
- The appearance of signs can now be changed by using a pen on them.
- Arcane tomes now have a new sprite, courtesy of /vg/ (not pictured: occasional eye flash):  
  maymayPlus a new attack animation (check the UI): 

)

### CODE REVIEW

**Comment by duncathan** [comment]
>signs constructable

:+1: SPACE JESUS IS SMILING DOWN ON US THROUGH OUR NONEXISTENT CEILING

**Comment by Xhuis** [comment]
Did a bit more work and resprited the tome to the one that /vg/ has because it looks nicer (may be reverted).

**Comment by Dorsisdwarf** [comment]
Make it the pipe painter or something or otherwise every sign on station will be turned into SINGULO WARNING ten seconds in.

File: code/game/objects/structures/signs.dm [hunk 1]
```diff
@@ -12,6 +17,78 @@
...
+		playsound(src, 'sound/items/Deconstruct.ogg', 50, 1)
+		user.visible_message("<span class='notice'>[user] unfastens [src].</span>", \
+							 "<span class='notice'>You unfasten [src].</span>")
+		new /obj/item/sign_backing(get_turf(user))
+		qdel(src)
+	if(istype(O, /obj/item/weapon/pen))
+		var/list/sign_types = list("Secure Area", "Biohazard", "High Voltage", "Radiation", "Hard Vacuum Ahead", "Disposal: Leads To Space", "Danger: Fire", "No Smoking", "Medbay", "Science", "Chemistry", \
+		"Hydroponics", "Xenobiology")
+		var/sign_type
+		switch(input(user, "Select a sign type.", "Sign Customization") as null|anything in sign_types)
```
Comment (RemieRichards) [inline code/game/objects/structures/signs.dm]: pls no.
make this an assoc list of ```"name" = a/type/path```

File: code/game/objects/structures/signs.dm [hunk 2]
```diff
@@ -12,6 +17,78 @@
...
+	desc = "A blank sign with adhesive backing."
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "backing"
+	w_class = 3
+	burn_state = 0 //Burnable, made of wood
+
+/obj/item/sign_backing/afterattack(atom/target, mob/user, proximity)
+	if(!isturf(target))
+		return ..()
+	if(!in_range(user,target))
```
Comment (RemieRichards) [inline code/game/objects/structures/signs.dm]: swap this for ```target.Adjacent(user)``` adjacency handles things ```in_range()```doesn't

File: code/game/objects/structures/signs.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/objects/structures/signs.dm]: I did this for multiple reasons, the first and foremost being that there are a ton of subtypes I would have to remove manually (barsigns...) so unless it's _absolutely_ necessary I'll probably keep it as is.

**Comment by Xhuis** [comment]
This is merge-ready unless anyone has any major gripes

**Pull request merged**